Cargo.toml

```diff
@@ -13,14 +13,17 @@ license = "Apache-2.0"
 bincode = "1"
 futures = "0.3"
 num_enum = "0.7"
-quinn = "0.10"
+quinn = { version = "0.11", features = ["ring"] }
```

Comment:
Isn't the `ring` feature needed for test only? If so, keep the quinn dependency without `ring` (`quinn = "0.11"`), and add the `ring` feature in dev-dependencies only. In that way, other crates depending on this crate won't have to build `ring`.

Reply:
I’ve created a new PR because the current issue was accidentally closed.